cmd/influxd/run/config.go

```diff
@@ -69,8 +69,10 @@ func NewConfig() *Config {
 	c.Monitor = monitor.NewConfig()
 	c.Subscriber = subscriber.NewConfig()
 	c.HTTPD = httpd.NewConfig()
+	c.Graphites = []graphite.Config{graphite.NewConfig()}
```

Comment:
It seemed like the correct place to put it since that's where all the other services are. The zero value for a bool is false so they will definitely be disabled by default. The changes can be moved to `NewDemoConfig()` if you want.